IDinsight/aaq-core
# Clean up migrations

- Main reviewer: @sidravi1 
- Time est: 15mins

## Clean up migrations files

## Ticket

No ticket - quick fix as discussed with @sidravi1 

## Description, Motivation and Context

Cleaned up the migration scripts and corrected the revision IDs to match correctly. Renamed them to follow an "action_name_object" format, like "create_user_queries_table".

## How Has This Been Tested?

- Run `make tests`. 
- Set up dev environment and run the main app, play around with `http://localhost:8000/docs` and check the database entries (using pgAdmin or elsewhere).

## To-do before merge

- [ ] Ensure PR #73 is merged

## Checklist:

This checklist is a useful reminder of small things that can easily be forgotten.
Put an `x` in all the items that apply and remove any items that are not relevant to this PR.
